tests/remote/remote_target.pm

```diff
@@ -1,34 +1,33 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2016-2018 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Remote ssh & vnc installation
-# Maintainer: mkravec <mkravec@suse.com>
+# Summary: Controller/master for remote installations
+# Tags: poo#9576
+# Maintainer: Martin Kravec <mkravec@suse.com>
 
 use base "opensusebasetest";
 use strict;
 use testapi;
 use lockapi;
 use mmapi;
+use mm_network;
 
 # poo#9576
 sub run {
-    # Notice MASTER system is ready for installation
-    assert_screen "remote_slave_ready", 200;
-    mutex_create "installation_ready";
-
-    # Wait until MASTER finishes installing the system
-    wait_for_children;
+    mutex_create("dhcp_client_configuration");
```

Comment:
Why do you need 2 mutexes here? - Why not just 'ssh_server_is_available'?
It seems that that controller job also waits for 2 consequent mutexes..

Reply:
dhcp_client_configuration has been removed.